### PR TITLE
Update travel fund for @ryzokuken

### PR DESCRIPTION
**Pull request by ryzokuken**
Keeping track of expenses and updating the travel fund, approved in https://github.com/nodejs/admin/pull/120 and inspired by https://github.com/nodejs/admin/pull/124

**NOTE:** The value for dollars changed in the last few days, updated the original values to lower ones.

| Item | Proposed | Actual | Actual (USD) | Comments |
|---|---|---|---|---|
| Airfare | $800 | ₹51161 | $770->$758.20 | Paid in full |
| Lodging | $400 | €345.5 | $420->$402.92 | Paid in full |
| Miscellaneous | $300 | | $222.83 | Paid in full |
| **Total** | $1500 | | $1384 | |

Breakdown for Miscellaneous

| Item | Actual | Actual (USD) | Comments |
|---|---|---|---|
| Insurance | €13 | $15.16 | |
| Visa | ₹2102+₹4900=₹7002 | $103.77 | VFS Fee + Visa Fee |
| Travel | €31.60+€42.50=€74.1 | $86.41 | Bahn + Cab |
| Phone | €15.00 | $17.49 | |
| **Total** | | $222.83 | |

### CODE REVIEW

**Comment by chowdhurian** [comment]
Adding tag at request of PR owner.

File: MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Richard Littauer | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin
 Dan Shaw | Spring 2018 Collaborator's Summit and Code & Learn | Attendance & WG facilitator | Berlin, Germany | May 30 - Jun 3 | $1500
 Benjamin Zaslavsky | Collab Summit Spring 2018 | Attendance - Inclusivity Initiative | Berlin, Germany | May 31 - Jun 1 2018 | $385.59
 Agiri Abraham JNR | Collab Summit Spring 2018 | Attendance - Inclusivity Initiative & Mentorship | Berlin, Germany | May 31 - Jun 3 2018 | $1765.00
-Ujjwal Sharma | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin, Germany | May 31-June 1 2018 | $1500
+Ujjwal Sharma | Collab Summit Spring 2018 | Attendance & Collaborate | Berlin, Germany | May 31-June 1 2018 | $1490
```
Comment (mhdawson) [inline MEMBER_TRAVEL_FUND.md]: This does not match the $1384 in the updated description

Comment (ryzokuken) [reply MEMBER_TRAVEL_FUND.md]: Oops, updating.

**Comment by ryzokuken** [comment]
@mhdawson done 😅

**Comment by Trott** [comment]
Needs a rebase. Can we remove the `do-not-merge` label at this point?

**Comment by ryzokuken** [comment]
@Trott yeah, definitely. Let me try resolving conflicts using GH's tool.

**Comment by ryzokuken** [comment]
@Trott rebased successfully and updated the dates, PTAL.

**Comment by ryzokuken** [comment]
@Trott yes, you should. Thanks.

**Pull request merged**